adafruit_tmp117.py

```diff
@@ -542,3 +470,10 @@ def _read_status(self) -> Tuple[int, int, int]:
 
     def _read_temperature(self) -> float:
         return self._raw_temperature * _TMP117_RESOLUTION
+
+    # pylint: disable=no-self-use
+    @staticmethod
```

Comment:
I think it would be better to have it at the top. We should put this in the temperature property, however, currently we used it to make one measure in the `__init__`, We do this, as if the sensor has not finished the conversion it will return `0x8000`. I will measure what is best for the memory and modify if necessary accordingly.

Reply:
pls see https://github.com/adafruit/Adafruit_CircuitPython_TMP117/pull/15/commits/719aebd412fec6bc7fb49101a2b4e055121270c2